lib/bolt/config/options.rb

```diff
@@ -475,6 +477,13 @@ module Options
           _plugin: true,
           _example: { "cleanup" => false, "tmpdir" => "/tmp/bolt" }
         },
+        "lxd" => {
+          description: "A map of configuration options for the LXD transport. The LXD transport is "\
+                       "experimental and may include breaking changes between minor versions.",
```

Comment:
```suggestion
                       "experimental and might include breaking changes between minor versions.",
```

Docs prefers 'might' as it's less ambiguous.